src/main.rs

```diff
@@ -48,10 +47,95 @@ fn main() {
     let db_path = dirs::db_path().to_str().unwrap().to_string();
     info!("Using DB at path {:?}", db_path);
 
+    let asset_path = get_asset_path();
+    info!("Using aw-webui assets at path {:?}", asset_path);
+
     let server_state = endpoints::ServerState {
         datastore: Mutex::new(datastore::Datastore::new(db_path)),
-        asset_path: PathBuf::from("aw-webui").join("dist"),
+        asset_path: asset_path,
     };
 
     endpoints::build_rocket(server_state, &config).launch();
 }
+
+use std::path::PathBuf;
+
+// The appdirs implementation of site_data_dir is broken on computers which has flatpak installed
```

Comment:
Found this issue when I ran on my laptop.
Kind of a shame that I had to add this, but it works.